go/ql/lib/ext/os.model.yml

```
@@ -40,3 +40,12 @@ extensions:
       - ["os", "", False, "ExpandEnv", "", "", "Argument[0]", "ReturnValue", "taint", "manual"]
       - ["os", "", False, "NewFile", "", "", "Argument[0]", "ReturnValue", "taint", "manual"]
       - ["os", "File", True, "Fd", "", "", "Argument[receiver]", "ReturnValue", "taint", "manual"]
+      - ["os", "File", True, "Read", "", "", "Argument[receiver]", "Argument[0].Element", "taint", "manual"]
+      - ["os", "File", True, "ReadAt", "", "", "Argument[receiver]", "Argument[0].Element", "taint", "manual"]
```

```suggestion
      - ["os", "File", True, "Read", "", "", "Argument[receiver]", "Argument[0]", "taint", "manual"]
      - ["os", "File", True, "ReadAt", "", "", "Argument[receiver]", "Argument[0]", "taint", "manual"]
```
We taint the whole `[]byte` rather than individual bytes, treating it like a string. (And it would be `ArrayElement` rather than `Element`. We don't use `Element` in Go at the moment.)